apps/src/constants.js

```diff
@@ -57,6 +58,7 @@ exports.TestResults = {
 
   // Codes for unvalidated levels.
   UNSUBMITTED_ATTEMPT: -50,           // Progress was saved without submitting for review, or was unsubmitted.
+  UNVALIDATED_SUBLEVEL: -49,          // Either a member of a LevelGroup or a contained level, for which the result has not been validated
```

Comment:
I think UNVALIDATED is actually good because it implies that it's not _yet_ validated.  Hopefully nobody mistakes it for INVALIDATED :)